specs/MultiProfile.md

```diff
@@ -217,6 +232,76 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(
 }
 ```
 
+### Manage password-autosave and general-autofill settings in profile
+
+```cpp
+HRESULT AppWindow::ManagePasswordAutosaveInProfile(ICoreWebView2Controller* controller)
+{
+    // ...
+
+    // Get the profile object.
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
+    Microsoft::WRL::ComPtr<ICoreWebView2Profile> webView2Profile;
+    CHECK_FAILURE(webView2->get_Profile(&webView2Profile));
+    
+    // Get current value of password-autosave property.
+    BOOL enabled;
+    CHECK_FAILURE(webView2Profile->get_IsPasswordAutosaveEnabled(&enabled));
+
+    // Set password-autosave property to the opposite value to current value.
+    if (enabled)
+    {
+        CHECK_FAILURE(webView2Profile->put_IsPasswordAutosaveEnabled(FALSE));
+        MessageBox(
+            nullptr, L"Password autosave will be disabled after next navigation in all WebView2 with the same profile.",
```

Comment:
Don't use nullptr as window parameter to MessageBox. Use the window that is hosting the webview. Otherwise, you get into bad reentrant states.

Reply:
Make sense, done, thanks.